### DISCUSSION TITLE
Simple selfhosted deployment

### DISCUSSION THREAD

**Discussion by gaby** [akvorado/akvorado#658] [Q&A] [answered]
I want to try this as a selfhosted solution, is it possible to have a more simple docker-compose file without geoip/traefik? Does akvorado work without sending SNMP/Geo data?

**Comment by vincentbernat** [comment, marked as answer]
You can remove GeoIP and Traefik if you want. Akvorado still needs SNMP access to the exporters.

**Reply by gaby** [reply]
Thanks for the info, I will have to figured out how to add snmp. My setup only has Sflow/Netflow right now.
